src/views/Layout.css

```diff
@@ -1,31 +1,23 @@
-/**
- * The `env()` function used in this block of code
- * allows us to reference special variables that might be defined
- * by certain browsers. In this case, we are making sure that
- * the app stays within the 'safe' viewable area of a mobile phone,
- * and isn't covered up by the phone's UI.
- *
- * @see: https://developer.mozilla.org/en-US/docs/Web/CSS/env
- */
 .Layout {
 	display: flex;
 	flex-direction: column;
 	margin: 0 auto;
 	max-width: 100vw;
+	height: 100vh;
 	padding-top: calc(1rem + env(safe-area-inset-top, 1rem));
 	padding-bottom: calc(1rem + env(safe-area-inset-bottom, 1rem));
-	width: 54rem;
+	width: 60rem;
 }
 
 .Layout-header {
-	background-color: var(--color-bg);
 	position: sticky;
```

Comment:
This style that was inherited with the repo makes this image slide over the list and content on the page when you scroll down. I would look into reworking the design to either remove the position sticky so the title stays in place or maybe putting a solid background behind the image that covers the text so there's not a clash between the logo text covering the list text

![positionSticky](https://user-images.githubusercontent.com/89806097/186318012-796ae969-b800-4728-abab-d0f208cf5519.PNG)

Reply:
Oh great catch! I've added a background color and it fixed it.